### PR TITLE
Fix/settings page

### PR DESCRIPTION
**Pull request by moraleida**
### 🎫 Ticket

[ET-1368] 

### ✔️ Checklist
- [ ] I've included a changelog entry in the readme.txt file. 
- [ ] My code is tested. 
- [ ] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ET-1368]: https://theeventscalendar.atlassian.net/browse/ET-1368?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Tickets/Commerce/Gateways/Stripe/Settings.php [hunk 1]
```diff
@@ -457,4 +457,33 @@ public function get_connection_settings_html() {
...
+	 * @return mixed
+	 */
+	public function reset_hidden_field_values( $value, $field_id, $field ) {
+
+		if ( $value ) {
+			return $value;
+		}
+
+		if ( $field_id === static::$option_checkout_element_payment_methods ) {
+			return self::DEFAULT_PAYMENT_ELEMENT_METHODS;
```
Comment (juanfra) [inline src/Tickets/Commerce/Gateways/Stripe/Settings.php:479]: I think we could improve this by getting the option value here and returning it. That way if someone changes their mind and changes the checkout method they still have the same configuration.

File: src/Tickets/Commerce/Gateways/Stripe/Settings.php [hunk 2]
```diff
@@ -457,4 +457,33 @@ public function get_connection_settings_html() {
...
+		}
+
+		if ( $field_id === static::$option_checkout_element_card_fields ) {
+			return self::COMPACT_CARD_ELEMENT_SLUG;
```
Comment (juanfra) [inline src/Tickets/Commerce/Gateways/Stripe/Settings.php:483]: I think we could improve this by getting the option value here and returning it. That way if someone changes their mind and changes the checkout method they still have the same configuration.

File: src/Tickets/Commerce/Gateways/Stripe/Settings.php (hunk 1 above)
Comment (moraleida) [reply src/Tickets/Commerce/Gateways/Stripe/Settings.php]: fixed

File: src/Tickets/Commerce/Gateways/Stripe/Settings.php (hunk 2 above)
Comment (moraleida) [reply src/Tickets/Commerce/Gateways/Stripe/Settings.php]: fixed

**Review by juanfra — APPROVED** [review]
🚀

**Pull request merged**